README.md

```diff
@@ -83,6 +83,25 @@ Inflates the `resources` from a `req` objects `body` into a collection of object
 const objs = require('azure-chaos-fn/parsers').resourcesToObjects(req)
 ```
 
+### auditer
+
+A documented implementation of the verbose logging format defined in [Auditing](https://github.com/Azure/platform-chaos/wiki/Auditing)
+
+Initialize the auditer by using the following method:
```

Comment:
we should talk more about what we're doing here - modifying context (and that it is azure specific) to help the user. Further we should clarify that there's some `index.auditDirectly` capability